cohere/client.py

```diff
@@ -150,7 +150,18 @@ def generate(
             truncate (str): (Optional) One of NONE|START|END, defaults to END. How the API handles text longer than the maximum token length.\
             stream (bool): Return streaming tokens.
         Returns:
-            a Generations object if stream=False, or a StreamingGenerations object if stream=True
+            if stream=False: a Generations object
+            if stream=True: a StreamingGenerations object including:
+                id (str): The id of the whole generation call
+                generations (Generations): same as the response when stream=False
+                finish_reason (string) possible values:
+                    COMPLETE: when the stream successfully completed
+                    ERROR: when an error occurred during streaming
+                    ERROR_TOXIC: when the stream was halted due to toxic output.
+                    ERROR_LIMIT: when the context is too big to generate.
+                    USER_CANCEL: when the user has closed the stream / cancelled the request
+                    MAX_TOKENS: when the max tokens limit was reached.
+                texts (List[str]): list of segments of text streamed back from the API
```

Comment:
Would you mind adding some examples to the docstring? Like [here](https://github.com/cohere-ai/cohere-python/blob/80a2571026e4c2ea1443855b83b933b91df244cc/cohere/client.py#L419)